### PR TITLE
[DO NOT MERGE]: DNM: Revert "[CVP-tmp-fix] Temporarily increase running count of 4.11-4.15 clusters to 5"

### PR DESCRIPTION
**Pull request by samvarankashyap**
Reverts openshift/release#44673

### CODE REVIEW

File: clusters/hive/pools/cvp/cvp-ocp-4-11-amd64-aws-eu-west-2_clusterpool.yaml [hunk 1]
```diff
@@ -24,16 +24,16 @@ spec:
...
   platform:
     aws:
       credentialsSecretRef:
         name: cvp-aws-credentials
       region: eu-west-2
   pullSecretRef:
     name: pull-secret
-  runningCount: 5
-  size: 10
+  runningCount: 1
```
Comment (nmars) [inline clusters/hive/pools/cvp/cvp-ocp-4-11-amd64-aws-eu-west-2_clusterpool.yaml:35]: Should this be `2` to be consistent with the other versions?

**Comment by samvarankashyap** [comment]
no longer applicable

**Pull request closed without merging**